deployments/polygon.staging.json

```
@@ -46,7 +46,10 @@
   "CelerCircleBridgeFacet": "0x371E073f6A09DCBEE1D2Ac56E940F878a0Ba9DAE",
   "HopFacetOptimized": "0xf82135385765f1324257ffF74489F16382EBBb8A",
   "SymbiosisFacet": "0x21571D628B0bCBeb954D5933A604eCac35bAF2c7",
-  "AcrossFacetV3": "0xe2e5428F972d9C0a5Ba433e0c402752b472dB248",
+  "AcrossFacetV3": "0xAd99305a4277B417d1711CB358Dcf124181Ec3C3",
+  "Permit2Proxy": "0x6FC01BC9Ff6Cdab694Ec8Ca41B21a2F04C8c37E5",
```

Biome (1.9.4)

[error] The key Permit2Proxy was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)